src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -432,19 +432,21 @@ DisableMencoderNoskip =
 # Default: "", i.e. UMS will not send such header information.
 SubtitleHttpHeader = 
 
-# This option adds black borders to the video to prevent the renderer from
-# displaying an incorrect aspect ratio.
+# If this is true, it will always output video at 16/9 aspect ratio to the renderer,
+# meaning that all transcoded videos with different aspect ratios will have black
+# bars added to the edges to make them 16/9. It doesn't work with 3D SBS video.
 # If some videos on your renderer looked either stretched or squashed, enabling
 # this setting may fix that.
-# This is only reliable when MediaInfo = true.
```

Comment:
Ok, I'm guessing the idea was that only MediaInfo could properly parse the resolution, but I'd think FFmpeg does that just fine as well.